templates/routing-ortools/main.py

```diff
@@ -0,0 +1,177 @@
+"""
+Adapted example of the vehicle routing problem with Google OR-Tools.
+"""
+
+import argparse
+import json
+import sys
+import time
+from typing import Any
+
+from ortools.constraint_solver import pywrapcp, routing_enums_pb2
+
+
+def main():
+    """Entry point for the template."""
+    parser = argparse.ArgumentParser(description="Solve problems with OR-Tools.")
```

Comment:
```suggestion
    parser = argparse.ArgumentParser(description="Solves VRPs with max distance constraint.")
```

Reply:
would not do that. we want to point out that it is OR-Tools. not that there is a distance constraint

Reply:
Ok, fair. The distance constraint is not so important, but "solve problems" sounds a little too generic, right?

Reply:
ok, updated